packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx

```diff
@@ -266,12 +266,25 @@ const DateFilterInputs = <T extends ConditionalRule = DateFilterRule>(
                         onChange={(
                             range: [Date | null, Date | null] | null,
                         ) => {
-                            if (range && range[0] && range[1]) {
+                            if (range && (range[0] || range[1])) {
                                 onChange({
                                     ...rule,
-                                    values: range.map((value) =>
-                                        formatDate(value, undefined, false),
-                                    ),
+                                    values: [
+                                        range[0]
+                                            ? formatDate(
+                                                  range[0],
+                                                  undefined,
+                                                  false,
+                                              )
+                                            : moment(range[1]).add(-1, 'days'),
```

Comment:
don't we need to pass the momentjs result to the formatDate() ? 
```
values: [
  formatDate( range[0] || moment(range[1]).add(-1, 'days'), undefined, false),
  formatDate( range[1] || moment(range[0]).add(1, 'days'), undefined, false),
]
```

Reply:
formatDate takes a `MomentInput` , which can be a `Date`  (range[0] ) or  `Moment` (  moment(range[1]).add(-1, 'days')) 

Are you suggesting to  do `moment(range[1]).add(-1, 'days').toDate()` to always use a Date here ?

Reply:
My concern is that the fallback value is not being formatted like the rest of the values with `formatDate()` and might cause some issues.